server/text_generation_server/layers/tensor_parallel.py

```diff
@@ -69,12 +69,12 @@ def load(config, prefix: str, weights):
 
         # GPTQ,AWQ,EETQ don't quantize heads (nor embeddings)
         if config.quantize in ["gptq", "awq", "eetq", "marlin"]:
-            quantize = None
+            pass
```

Comment:
This feels wrong. We really want to set quantize=None in each of these I think.

Reply:
this is a lint for `unused-variable` as `quantize` is set but never used in this function.

reverted the change but i'm not sure if this check is needed..

@danieldk is there a case we need the `quantize` variable?